packages/make-styles/src/runtime/resolveStyleRules.ts

```diff
@@ -87,16 +88,31 @@ export function resolveStyleRules(
 
         resolveStyleRules(value, unstable_cssPriority, pseudo, media, combinedSupportQuery, result);
       } else if (property === 'animationName') {
-        // TODO: support RTL!
-        const keyframe = compileKeyframeRule(value);
-        const animationName = HASH_PREFIX + hashString(keyframe);
+        const animationNames = Array.isArray(value) ? value : [value];
```

Comment:
⚠️ I think that we should also handle a scenario when `animationName` is a string

Reply:
Oops, I see we will handle it in a previous condition. Can you please a unit test for this case?